cockpit/src/main/java/com/polidea/cockpit/utils/FileUtils.kt

```diff
@@ -35,13 +36,24 @@ object FileUtils {
         }
 
         val savedCockpit = File(savedCockpitFilePath)
-        val list: Map<String, Any> = yaml.load(savedCockpit.bufferedReader().use {
+        val savedParamsMap: Map<String, Any> = yaml.load(savedCockpit.bufferedReader().use {
             it.readText()
         })
 
-        list.forEach {
-            CockpitManager.addParam(CockpitParam(it.key, it.value.javaClass, it.value))
+        val inputMap: Map<String, Any> = inputParamsProvider.getInputParams()
+
+        savedParamsMap.forEach {
+            if (inputMap.contains(it.key)) {
+                CockpitManager.addParam(CockpitParam(it.key, it.value.javaClass, it.value))
+            }
         }
     }
+}
 
+class InputParamsProvider {
+    fun getInputParams(): Map<String, Any> {
+        return Yaml(LoaderOptions()).load(File("src/main/assets/cockpit.yml").bufferedReader().use {
```

Comment:
We can't access assets in this way. Assets can be accessible using `AssetManager`. We have to pass `AssetManager` instance to `InputParamsProvider` and change `File("src/main/assets/cockpit.yml")` to `assets.open("cockpit.yml")`, where `assets` is `AssetManager`